### PR TITLE
Add canvasContextOptions API for use by the getContext

### LINKED ISSUE #760: Include the willReadFrequently in getContext
Hello Team,

We are trying to fix an issue with signature_pad in our project where the initial value was not set properly. After a bit of research we made it fixed by using `getImageData`, and `putImageData`. We also noticed that calling the `getContext` with `willReadFrequently` enabled on our side would raise a warning in the browser. Further investigation led us to the conclusion that we also need to modify the very first call to `getContext`, which is done by signature_pad. After that, the warning disappeared.

Link to our project https://github.com/Megabit/Blazorise/pull/5349

---

If you agree to the change, we can create a PR to include that change in here.

### PR DESCRIPTION
**Pull request by stsrki**
Closes #760

### CODE REVIEW

**Comment by stsrki** [comment]
I added the `willReadFrequently` option to be passed to the `canvas.getContext`. Since the name `willReadFrequently` might be too generic, I propose renaming it to

- `willReadCanvasFrequently`, or
- `willReadContextFrequently`, or
- `willReadCanvasContextFrequently`

**Comment by UziTech** [comment]
I think it would be better to have a contextOptions instead of each option. That way when Chrome adds more options we won't need to change anything.

**Comment by stsrki** [comment]
> I think it would be better to have a canvasContextOptions instead of each option. That way when Chrome adds more options we won't need to change anything.

Makes sense. I will make the changes.

**Comment by stsrki** [comment]
I have update the API and add canvasContextOptions.

File: package-lock.json
Comment (UziTech) [inline package-lock.json:1]: Delete this file and use yarn

File: README.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ signaturePad.on();
 <dd>(string) Color used to draw the lines. Can be any color format accepted by <code>context.fillStyle</code>. Defaults to <code>"black"</code>.</dd>
 <dt>velocityFilterWeight</dt>
 <dd>(float) Weight used to modify new velocity based on the previous velocity. Defaults to <code>0.7</code>.</dd>
+<dt>canvasContextOptions</dt>
+<dd>(CanvasRenderingContext2DSettings) part of the Canvas API, provides the 2D rendering context for the drawing surface of a <code>canvas</code> element. It is used for drawing shapes, text, images, and other objects.</dd>
```
Comment (UziTech) [inline README.md:109]: can we link to MDN or something to show which options are available?

**Review by UziTech — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stsrki) [reply README.md]: Done.

**Review by UziTech — APPROVED** [review]
Thanks 💯

**Comment by stsrki** [comment]
Thank you for merging this feature. Much appreciated 🙏

**Pull request merged**